src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -275,14 +277,27 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     });
                 }
 
+                if (string.Equals(_request.Project.RestoreMetadata?.RestoreAuditProperties?.EnableAudit, "enable", StringComparison.InvariantCultureIgnoreCase))
+                {
+                    telemetry.StartIntervalMeasure();
+                    AuditUtility audit = new AuditUtility(
```

Comment:
Why?  While I don't usually agree with "don't use var", at least the theoretical benefit is to improve understanding of what datatype the reader might not know. But I don't understand the benefit of using implicit type, especially when the type has a short name.

Reply:
Is it different than this example in # 10 of https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style?

> The following are incorrect: `List<Fruit> fruits = new List<Fruit>();`
> the following are correct: `var fruits = new List<Fruit>();`

Reply:
If the only reason to change the code is because of the coding guidelines say to do it this way, without justifying why it benefits the repo, then the coding guidelines document probably should be changed.

Some of the guidelines help reduce churn by being opinionated about whitespace, so people don't keep changing whitespace between PRs. Some guidelines help reduce risk of bugs. Some help improve readability. But I don't feel like the "don't use var" guideline does any of those things

Reply:
If you feel our coding guidelines don't benefit our repo, then certainly propose your changes and rationale for that. However, since they're in place now, I think we should follow them, just as we ask our community members to do.